packages/meteor-accounts-saml/saml_server.js

```diff
@@ -75,14 +79,14 @@ Accounts.registerLoginHandler(function (loginRequest) {
 
 	var loginResult = Accounts.saml.retrieveCredential(loginRequest.credentialToken);
 	if (Accounts.saml.settings.debug) {
-		console.log("RESULT :" + JSON.stringify(loginResult));
+		console.log('RESULT :' + JSON.stringify(loginResult));
 	}
 
-	if (loginResult == undefined) {
+	if (loginResult === undefined) {
 		return {
-			type: "saml",
-			error: new Meteor.Error(Accounts.LoginCancelledError.numericError, "No matching login attempt found")
-		}
+			type: 'saml',
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Meteor" is not defined.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2313313354/issues/source?bid=3199634&fileBranchId=3199636#l88)